nats/js/client.py

```diff
@@ -922,7 +922,7 @@ async def get_last_msg(
         req_subject = f"{self._prefix}.STREAM.MSG.GET.{stream_name}"
         req = {'last_by_subj': subject}
         data = json.dumps(req)
-        resp = await self._api_request(req_subject, data.encode())
+        resp = await self._api_request(req_subject, data.encode(), timeout=self._timeout)
```

Comment:
Formatter from CI suggesting something like this:

```suggestion
        resp = await self._api_request(
            req_subject, data.encode(), timeout=self._timeout
        )
```